### PR TITLE
Added hint for using experimental noetic-devel branch.

### PR DESCRIPTION
**Pull request by mogumbo**
## Linked Issues:
| EPIC ⚡ | [OCEANWATER-601](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-601) |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-577](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-577) |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ setting up the rest of the OceanWATERS repositories:
 ## Getting Started
 - [Install software prerequisites](oceanwaters/doc/setup_dev_env.md)
 - [Download and build OceanWATERS](oceanwaters/doc/setup_oceanwaters.md)
+  - Default build instruction use ROS Melodic, but you will also find an option
+    to use experimental ROS Noetic support.
```
Comment (Samahu) [inline README.md:32]: The requirements page state that you need to use Ubuntu 18.04 and melodic. The note needs to proceed the requirements page in my opinion. I just moved your note one level up but consider rephrasing it to fit the new location.
```suggestion
## Getting Started
Default build instruction use ROS Melodic, but you will also find an option to use experimental ROS Noetic support.
- [Install software prerequisites](oceanwaters/doc/setup_dev_env.md)
- [Download and build OceanWATERS](oceanwaters/doc/setup_oceanwaters.md)
```

**Review by Samahu — CHANGES_REQUESTED** [review]
I think it makes sense to add the hint before the requirements and setup .. open to disputes.

**Comment by kmdalal** [comment]
> I think it makes sense to add the hint before the requirements and setup .. open to disputes.

I agree, and I think we should further and mention melodic-devel, since we're mentioning noetic-devel.

**Comment by mogumbo** [comment]
> > I think it makes sense to add the hint before the requirements and setup .. open to disputes.
> 
> I agree, and I think we should further mention melodic-devel, since we're mentioning noetic-devel.

done

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ setting up the rest of the OceanWATERS repositories:
 - [irg_open](https://github.com/nasa/irg_open)
 
 ## Getting Started
+Default build instruction use ROS Melodic, but you will also find an option to use experimental ROS Noetic support.
```
Comment (Samahu) [inline README.md:29]: [optional] formatting, add bold or italic to highlight the ROS Distro
```suggestion
Default build instruction use _ROS Melodic_, but you will also find an option to use experimental _ROS Noetic_ support.
```

**Review by Samahu — APPROVED** [review]

**Review by kmdalal — APPROVED** [review]

**Pull request merged**